pyaedt/edb_core/components.py

```diff
@@ -1287,20 +1287,28 @@ def disable_rlc_component(self, component_name):
         return False
 
     @pyaedt_function_handler()
-    def set_solder_ball(self, component="", sball_diam="100um", sball_height="150um"):
+    def set_solder_ball(
+        self, component="", shape="Cylinder", sball_diam="100um", sball_height="150um", sball_mid_diam=None
+    ):
```

Comment:
@ring630 I would just add the new parameters at the end.
```suggestion
    def set_solder_ball(
        self, component="", sball_diam="100um", sball_height="150um", shape="Cylinder", sball_mid_diam=None
    ):
```

Reply:
Thanks for the comment.